framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -923,6 +931,25 @@ public AnnotationFormatter getAnnotationFormatter() {
         return Collections.unmodifiableSet(supportedQuals);
     }
 
+    /**
+     * Returns an immutable set of the type qualifiers supported by this checker.
```

Comment:
Here and in the `@return` it would be better to use something closer to the documentation in the field, like `Returns an immutable set of the fully-qualified names of the classes supported by this checker.`
At the moment this doesn't explain what the Strings represent.